doc/docs/sample-replidraw.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Replidraw
+slug: /samples/replidraw
+---
+
+A tiny Figma-like multiplayer graphics editor.
+
+Built with [Replicache](https://replicache.dev), [Next.js](https://nextjs.org/),
+[Fauna](https://fauna.com/), and [React
+Designer](https://react-designer.github.io/react-designer/).
+
+Running live at https://replidraw.vercel.app/.
```

Comment:
@aboodman Do you have a copy of the Replidraw movie you posted to Twitter?

Reply:
You can get it here: https://video.twimg.com/ext_tw_video/1387829808222986241/pu/vid/1280x720/JvnqTzYPhf6Xjjp9.mp4?tag=12

Reply:
The file is pretty large. I'm just skipping the image for now